{
    "config": {
        "abort": {
            "already_configured": "Account is already configured",
            "cannot_connect": "Failed to connect",
            "reauth_successful": "Re-authentication was successful",
            "unknown": "Unexpected error"
        },
        "error": {
            "cannot_connect": "Failed to connect",
            "invalid_auth": "Invalid authentication",
            "unknown": "Unexpected error"
        },
        "step": {
            "reauth": {
                "data": {
                    "password": "Password",
                    "username": "Username"
                }
            },
            "user": {
                "data": {
                    "password": "Password",
                    "username": "Username"
                }
            }
        }
    }
}